api/PayrollProcessor.Api/Properties/launchSettings.json

```diff
@@ -19,12 +19,10 @@
     },
     "PayrollProcessor.Api": {
       "commandName": "Project",
-      "launchBrowser": true,
-      "launchUrl": "risk",
       "environmentVariables": {
         "ASPNETCORE_ENVIRONMENT": "Development"
       },
-      "applicationUrl": "https://localhost:5001;http://localhost:5000"
```

Comment:
We can add back https bindings for all the projects (and update configuration) - this change was made to get things working.